code/processes/tailreader.q

```diff
@@ -7,6 +7,9 @@ currentpartition:@[value;`currentpartition;getpartition[]]
 basedir:raze (getenv`KDBTAIL),"/tailer",(string .ds.segmentid),"/"         /-define associated tailer base directory
 taildir:`$ basedir,string currentpartition;                                /-define tailDB direction
 
+/- log message if tailreader is started without datastriping activated
+if[not .ds.datastripe;.lg.o[`load;"Datastriping is disabled: please verify whether ",(string .proc.procname)," process should be running."]];
```

Comment:
An out log might get lost/ignored in the vast amount of other out logs TorQ produces, unless someone is looking specifically for it (and if someone knows to look for this particular log, they probably also know not there's no point having a tailreader without datastriping).

Should we just log an error (which will cause the process to exit) to make it a bit more obvious for people who aren't looking?

Reply:
Good point: also maintains parity with a similar `.lg.e` message in the tailer script.